libs/components/manifest/package.json

```diff
@@ -28,5 +26,7 @@
     "./public-api.json": {
       "default": "./public-api.json"
     }
-  }
+  },
+  "type": "commonjs",
```

Comment:
These changes were automatically applied after running the `npm run dev:update-library-dependencies` command. 🤷